experiments/explore-bigdata-to-smalldata/find_aff_id.py

```
@@ -0,0 +1,68 @@
+schema = pq.read_schema('./sciscinet_papers.parquet')
+print("Схема таблицы:")
+print(schema)
+
+INSTITUTION_ID = "I172901346"  # ваш ID
+
+# Шаг 1: достаём paper_id из локального файла
+paper_ids = con.execute(f"""
+    SELECT *
+    FROM read_parquet('./data/sciscinet_paper_author_affiliation.parquet')
+    WHERE institutionid = '{INSTITUTION_ID}'
+""").fetchdf()
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)